views/admin/new_event.erb

```diff
@@ -113,6 +113,25 @@
                 </div>
                 <br>
 
+                <div class="dcf-form-group dcf-input-group">
+			        <div class="dcf-input-checkbox">
+			            <input type="checkbox" <%= 'checked="checked"' if event.has_authorization %> id="authorize_tools_checkbox" name="authorize_tools_checkbox">
```

Comment:
I think the authorize tools checkbox should probably be checked automatically if the user is creating a new event from a preset. This would just prevent them from accidentally forgetting to check it. Could change this logic to ```<%= 'checked="checked"' if event.has_authorization || (event.id.nil? && !preset_event.nil?) %>``` to achieve this